lib/json_api_resource/resource.rb

```diff
@@ -1,3 +1,12 @@
+require 'active_support'
+require 'active_support/concern'
+require 'active_support/core_ext/module'
+require 'active_support/core_ext/class/attribute'
+require 'active_model'
+require 'active_model/model'
+require 'active_model/validations'
+require 'active_model/callbacks'
+
 module JsonApiResource
   class Resource
     include ActiveModel::Model
```

Comment:
Then they can open an issue or MR. Concerns and am::m are things we rely heavily on